layout/templates/simple.mustache

```diff
@@ -1,4 +1,10 @@
 <div class="container">
+    <div class="title">
+      <h3>How much water do we use? U.S. water withdrawals, 1950-present The USGS National Water-Use Science Project has documented how and where we use water for the last 65 years.</h3>
```

Comment:
idk, whatever you guys want to do, not a yaml guy so if you need it in there I say go for it.